File: src/main/scala/li/cil/oc/integration/buildcraft/ModBuildCraft.scala
package li.cil.oc.integration.buildcraft

import li.cil.oc.api.Driver
import li.cil.oc.integration.ModProxy
import li.cil.oc.integration.Mods

object ModBuildCraft extends ModProxy {
  override def getMod = Mods.BuildCraft

  override def initialize() {
    Driver.add(new DriverPipeTE)
    Driver.add(new DriverPowerReceptor)
    Driver.add(new DriverMachine)
  }
}

File: src/main/scala/li/cil/oc/integration/buildcraft/ModBuildCraft.java
package li.cil.oc.integration.buildcraft;

import li.cil.oc.api.Driver;
import li.cil.oc.integration.IMod;
import li.cil.oc.util.mods.Mods;

public final class ModBuildCraft implements IMod {
    @Override
    public Mods.Mod getMod() {
        return Mods.BuildCraft();
    }

    @Override
    public void initialize() {
        Driver.add(new DriverPipeTE());
        Driver.add(new DriverPowerReceptor());
        Driver.add(new DriverMachine());
    }
}
